### ISSUE TITLE
No   Highlighting choice in the right-Click  With Buster Xfce4 and Gtk3

### ISSUE THREAD

**Issue by al-louarn** [IgnorantGuru/spacefm#764]
Hello, I enjoy SpaceFM (1.0.6) since a few days, but in the right-click-menu in the panel (neither in the path bar) I don't have item hightlighting. And  in the manual I don't saw anyway to get it without 
right-click.
May be unrelated, but I only found an option for wallpaper png or  color, but it doesn't work.
  I wonder what I missed.  Or could  Xfce4-config or default themes  take precedence ?
Thanks for SpaceFM

**Comment by ncmprhnsbl** [comment]
not sure what you mean... do you mean the right-click menu for customizing a menu/panel item?

as for the wallpaper, i assume you're running xfce4 DE, so all desktop functions including wallpaper will be handled by xfce4 settings and the desktop settings within spacefm will have no effect.

**Comment by al-louarn** [comment]
Whoo ! So  faster !!!
Great thanks
 for xfce I agree, I think it would be better without Xfce, but it's my first try.
But yes  I do mean "the right-click menu for customizing a panel"  to obtain highlighting in the current panel or current path bar is not efficient.
This is explained in the [spacefm-manual](http://ignorantguru.github.io/spacefm/spacefm-manual-en.html#gui-pan-foc).
Other example with "columns to appear" (always with right-click in the panel ) : in the left panel my choices are respected and I can see type of files, or last-date-access, but in the wright panel I can only see the list of files or directories, never other column.
Possible it was the width of the fisrt columns but I cannot reduce it with the mouse.  
I saw that the  ~/.config/SpaceFM/session is not writable.
So I could backup it and  try with a new one...

**Comment by al-louarn** [comment]
I wrote : " Possible it was the width of the fisrt columns but I cannot reduce it with the mouse."
At night, take advice ! 
I have backuped the /.config/session/ files, then reloaded SpaceFM  and try to modify it appaerance.
All works.
For highlighting the focused panel I was wrong, It works by right-click on the STATUS bar not for the panel neither the path bar. Then Only the status bar is highlighted. That's enough

Sorry I asked you for nothing.
